src/_data/sites/webperformance-report.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://webperformancereport.com/",
+	"name": "WebPerformance Report",
+	"description": "Web performance report every week in your inbox. Measure, Monitor and Improve.",
+	"twitter": "@WebPerfReport",
+	"source_url": "https://github.com/edwinmh/webperformance-report"
```

Comment:
@edwinmh Is this an Eleventy site? I don't see an .eleventy.js config file (unless it's using all the default values) or a package.json file.

Reply:
Hi @pdehaan! Yes, it is an Eleventy site. The only thing is that I upload the _site folder to the prod server. In the repo you can see that I work with 11ty.

Reply:
Yep, I see the `url` filter in the layouts, cool!